src/pages/signin/ResendValidationForm.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import {Text, View} from 'react-native';
+import styles from '../../styles/styles';
+import SubmitButton from './SubmitButton';
+import {resendValidationLink} from '../../libs/actions/Session';
+
+class ResendValidationForm extends React.Component {
+    constructor() {
+        super();
+
+        this.validateAndSubmitForm = this.validateAndSubmitForm.bind(this);
+
+        this.state = {
+            formSuccess: '',
+            isLoading: false,
+        };
+    }
+
+    /**
+     * Check that all the form fields are valid, then trigger the submit callback
+     */
+    validateAndSubmitForm() {
+        this.setState({
+            formSuccess: 'Link has been resent',
+        });
+
+        resendValidationLink();
+
+        setTimeout(() => {
+            this.setState({formSuccess: ''});
+        }, 5000);
```

Comment:
Why is this necessary?

Reply:
I just didn't like the success message staying there forever. I think there is a diminishing return for a success message remaining visible

Reply:
Didn't quite understand your meaning here. Why does it matter if it disappears?

Reply:
> Why does it matter if it disappears?

I'll turn the question back around (since I also don't understand why you asked the original question). Why does this matter if it disappears after 5 seconds?

Reply:
I asked this because I didn't understand _why_ we are doing it. And it would be nice to have a comment in the code to explain why.